charts/descheduler/templates/clusterrole.yaml

```diff
@@ -12,6 +12,11 @@ rules:
 - apiGroups: [""]
   resources: ["nodes"]
   verbs: ["get", "watch", "list"]
+{{- if .Values.deschedulerPolicy.strategies.RemovePodsViolatingTopologySpreadConstraint.enabled }}
```

Comment:
In my opinion the conditional is not needed. The namespace list permission can just always be included. It is already included in file `kubernetes/base/rbac.yaml`.

Reply:
https://github.com/kubernetes-sigs/descheduler/blob/70d1fadae750207acd6f7f3db07f868c2624b368/kubernetes/base/rbac.yaml#L13-L15

Reply:
sounds good to me, updating